integrations/overview.mdx

```diff
@@ -0,0 +1,30 @@
+Benji Integrations allow you to connect your payment processor to Benji, enabling seamless loyalty and rewards experiences for your customers.
+
+## Supported Integrations
+| Platform | Status        |
+|----------|--------------|
+| Square   | Available    |
```

Comment:
Can we link the Available tag to the actual Square documentation?

Reply:
Yep, will add!